lib/Service/RelatedService.php

```diff
@@ -549,6 +550,14 @@ private function getRelatedResourceProviders(): array {
 			}
 		}
 
+		if ($this->appManager->isInstalled('groupfolders')) {
+			try {
+				$providers[] = Server::get(GroupFoldersRelatedResourceProvider::class);
+			} catch (NotFoundExceptionInterface | ContainerExceptionInterface $e) {
+				$this->logger->notice($e->getMessage());
```

Comment:
This will log a notice if the app is not installed? A bit weird since it’s not a hard dependency no?

Reply:
It’s the same for all other apps so leaving this at notice.